docs/source/guides/building_bentos.rst

```diff
@@ -232,8 +232,79 @@ For example, with NLP projects you can preinstall NLTK data with:
++------------------+-------------------+---------------+-------------+
+|                  | requirements_txt  |               |             |
++------------------+-------------------+---------------+-------------+
+|                  | packages          |               |             |
++------------------+-------------------+---------------+-------------+
+|                  | lock_packages     |               |             |
++------------------+-------------------+---------------+-------------+
+|                  | index_url         |               |             |
++------------------+-------------------+---------------+-------------+
+|                  | no_index          |               |             |
++------------------+-------------------+---------------+-------------+
+|                  | trusted_host      |               |             |
++------------------+-------------------+---------------+-------------+
+|                  | find_links        |               |             |
++------------------+-------------------+---------------+-------------+
+|                  | extra_index_url   |               |             |
++------------------+-------------------+---------------+-------------+
+|                  | pip_args          |               |             |
++------------------+-------------------+---------------+-------------+
+|                  | wheels            |               |             |
++------------------+-------------------+---------------+-------------+
+| BentoBuildConfig |                   |               |             |
++------------------+-------------------+---------------+-------------+
+|                  | service           |               |             |
++------------------+-------------------+---------------+-------------+
+|                  | description       |               |             |
++------------------+-------------------+---------------+-------------+
+|                  | labels            |               |             |
++------------------+-------------------+---------------+-------------+
+|                  | include           |               |             |
++------------------+-------------------+---------------+-------------+
+|                  | exclude           |               |             |
++------------------+-------------------+---------------+-------------+
+|                  | additional_models |               |             |
++------------------+-------------------+---------------+-------------+
+|                  | docker            |               |             |
++------------------+-------------------+---------------+-------------+
+|                  | python            |               |             |
++------------------+-------------------+---------------+-------------+
+|                  | conda             |               |             |
++------------------+-------------------+---------------+-------------+
```

Comment:
Lets split each options to each table, and then for each options have a small description what they are